ansible/veos_vtb

```diff
@@ -54,6 +54,8 @@ all:
     sonic:
       vars:
         mgmt_subnet_mask_length: 24
+        ansible_connection: multi_passwd_ssh
+        ansible_altpassword: YourPaSsWoRd
```

Comment:
This should fix the `multi_passwd_ssh` is not accessible if `tests/veos_vtb` is selected as the inventory file. I think a fundamental problem is that the whole `ansible/group_vars` directory is not accessible when using `tests/veos_vtb` as the inventory file. I am not sure if any other things in `ansible/group_vars` are useful (or potentially useful) in pytest? If so, it is probably better to help pytest finding the directory (maybe a symbolic link)?

Reply:
If the directory `ansible/group_vars` is not needed, I guess we do not need to put these variables there anymore.

Reply:
This looks like ansible always looks for `group_vars` and `host_vars` under the same directory of inventory file. Probably we can try:
```
cd tests
pytest -i ../ansible/veos_vtb ...
```

Reply:
Yes, but this is not the case in our public Jenkins vtestbed jobs, which uses the symbolic link under `tests/`.